importexport/musicxml/exportxml.cpp

```diff
@@ -578,6 +578,22 @@ static QString color2xml(const Element* el)
             return "";
       }
 
+//---------------------------------------------------------
+//   fontStyleToXML
+//---------------------------------------------------------
+
+static QString fontStyleToXML(const FontStyle style, bool underline = true)
+      {
+      QString res;
+      if (style & FontStyle::Bold)
+            res += " font-weight=\"bold\"";
+      if (style & FontStyle::Italic)
+            res += " font-style=\"italic\"";
+      if (underline && style & FontStyle::Underline)
```

Comment:
that is, underline is governed by both a property in **style** and a boolean flag? Why is one thing not enough?

Reply:
as per musicxml documentation [here](https://usermanuals.musicxml.com/MusicXML/Content/EL-MusicXML-fingering_1.htm), fingering do support underline so we are overriding for that particular case using the underline parameter as `false`

Reply:
Suggest renaming "underline" to "allowUnderline" or "underlineIsAllowed"